### PR TITLE
spyhackerz.org

### PR DESCRIPTION
**Pull request by gork7777**
issue: #88574

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -47,6 +47,7 @@ fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfi
 !
 ||paraanaliz.com/wp-content/uploads/*/gedik_gif_banner_youtube.gif
 odatv4.com###menu-icon ~ div[style^="margin-top: "]
+spyhackerz.org##a[href="https://xleet.to/?ref=spy_banner"]
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:50]: Just `||i.ibb.co/*/ezgif-com-gif-maker-`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (gork7777) [reply TurkishFilter/sections/specific.txt:50]: What is the difference between them?

Comment (zloyden) [reply TurkishFilter/sections/specific.txt:50]: In this case, it's better to block this gif with size 1.1 Mb.

Comment (gork7777) [reply TurkishFilter/sections/specific.txt:50]: There are two types gif I guess. One is your rule, two is: `i.ibb.co/6HFswqX/900x300.gif`
We can block it like this? `||i.ibb.co/*/900x300.gif`
Second gif is occurs when the page is refreshed several times

Comment (zloyden) [reply TurkishFilter/sections/specific.txt:50]: Yes, also add a modifier `$domain=spyhackerz.org` to those rules.

**Review by zloyden — APPROVED** [review]

**Pull request merged**